src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/CreateControlledResourceFlight.java

```diff
@@ -17,6 +22,15 @@
  * depend on the resource type. The latter must be passed in via the input parameters map with keys
  */
 public class CreateControlledResourceFlight extends Flight {
+  /**
+   * Retry rule for Notebook GCP steps. If GCP is down, we don't know when it will be back, so don't
+   * wait forever.
+   */
+  private static final RetryRule NOTEBOOK_GCP_RETRY_RULE =
+      new RetryRuleExponentialBackoff(
+          /* initialIntervalSeconds= */ 1,
+          /* maxIntervalSeconds= */ 8,
+          /* maxOperationTimeSeconds= */ 30);
```

Comment:
is this a long enough maximum timeout? from the PR description I'd expect notebook description to take O(minutes)